quickwit/quickwit-ingest/src/ingest_v2/mod.rs

```diff
@@ -87,6 +87,29 @@ pub fn get_idle_shard_timeout() -> Duration {
         .unwrap_or(DEFAULT_IDLE_SHARD_TIMEOUT)
 }
 
+const INGEST_ROUTER_BUFFER_SIZE_ENV_KEY: &str = "QW_INGEST_ROUTER_BUFFER_SIZE_BYTES";
+
+const DEFAULT_INGEST_ROUTER_BUFFER_SIZE: u64 = if cfg!(test) { 8 } else { 256 }; // 256 MiB
```

Comment:
```suggestion
const DEFAULT_INGEST_ROUTER_BUFFER_SIZE_MB: u64 = if cfg!(test) { 8 } else { 256 }; // 256 MiB
```

Or use one of the byte unit crate thing.